oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticIndexWriter.java

```diff
@@ -114,6 +114,10 @@ private BulkProcessor initBulkProcessor() {
 
     @Override
     public void updateDocument(String path, ElasticDocument doc) {
+        if (!elasticConnection.isConnected()) {
```

Comment:
👎 we ping elastic on every single update/delete. This would kill indexing performance.

Reply:
Yes. This also leads to too many logs when ES is not available. Also, just catching and logging the exception if ES is unreachable again leads to too many errors in logs. Should we cache the result for sometime and check only if after that time?